src/Equinox.Core/Retry.fs

```diff
@@ -9,7 +9,9 @@ module Retry =
     /// Wraps an async computation in a retry loop, passing the (1-based) count into the computation and,
     ///   (until `attempts` exhausted) on an exception matching the `filter`, waiting for the timespan chosen by `backoff` before retrying
     let withBackoff (maxAttempts : int) (backoff : int -> System.TimeSpan option) (f : int -> Async<'a>) : Async<'a> =
-        if maxAttempts < 1 then raise (invalidArg "maxAttempts" "Should be >= 1")
+        if maxAttempts < 1 then 
```

Comment:
I do one liner guards like these as one liners intentionally - I believe that's normal esp if you have a bunch of nullArg etc. checks etc